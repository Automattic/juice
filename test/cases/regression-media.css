a,
p {
  color: green;
}

@media only screen and (max-width: 600px) {
  a { color: red; }
}