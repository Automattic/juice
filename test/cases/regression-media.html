<a href="#">Test</a>
<p>Test</p>
