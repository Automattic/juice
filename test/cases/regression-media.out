<html><body><a href="#" style="color: green;">Test</a>
<p style="color: green;">Test</p>
</body></html>